### DISCUSSION TITLE
Getting Started with TensorFlow Lite on Seeed Studio XIAO nRF52840 Sense | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1011] [Q&A]
# Getting Started with TensorFlow Lite on Seeed Studio XIAO nRF52840 Sense | Seeed Studio Wiki

Getting Started

https://wiki.seeedstudio.com/XIAO-BLE-Sense-TFLite-Getting-Started/

**Comment by tianning2001** [comment]
In file included from c:\Users\tiann\Documents\Arduino\libraries\Arduino_TensorFlowLite\src/TensorFlowLite.h:22:0,
                 from C:\Users\tiann\OneDrive\文档\Arduino\libraries\Seeed_Arduino_LSM6DS3-master\examples\IMU_Classifier\IMU_Classifier.ino:4:
c:\Users\tiann\Documents\Arduino\libraries\Arduino_TensorFlowLite\src/peripherals/peripherals.h:61:2: error: #error "unsupported board"
 #error "unsupported board"
  ^~~~~
exit status 1

Compilation error: exit status 1
